homeworks/andrii.tkachenko_Hodr1337/5-arrays/arrays.js

```diff
@@ -0,0 +1,31 @@
+function elka(height) {
```

Comment:
The function name should have a verb. For example, `updateUser`, `fetchEntity`, `findLowestNumber`, `showTicket`, `displayName`.

`elka` is a bad name for a function, because it doesn't describe what the function does and there is no verb in the name. Please, find a new one.